db/migrate/20230613201911_create_posts.rb

```diff
@@ -0,0 +1,21 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :AuthorId
```

Comment:
- [ ] Great job so far. We still have some work to do. You have not added indexes for foreign keys. As per project requirements: `All columns that are foreign keys should have a corresponding index.` We have foreign keys in thr posts, comments and likes table. Kindly add the indexes of the respective foreign keys.
![image](https://github.com/byiringiroscar/wikipedia-project-with-rails/assets/62849872/1348503c-85d4-4c8e-b577-cc98799e6f01)